lockup/src/lib.rs

```diff
@@ -315,8 +313,8 @@ mod tests {
                     vesting_schedule,
                     salt: SALT.to_vec().into(),
                 }
-                .hash()
-                .into(),
+                    .hash()
```

Comment:
I'm not sure, is this formatting right?

Reply:
CLion fixes it that way, and I've seen other places with the same formatting, so I decided to have these changes